Musical Collection/Musical Collection Console App/Classes/MusicalCollection.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Musical_Collection_Console_App.Interfaces.Classes_Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Musical_Collection_Console_App.Classes
+{
+    public abstract class MusicalCollection : IMusicalCollection
+    {
+        private string _name;
+        private List<Song> _collection;
+        private List<string> _genres;
+        public MusicalCollection(string name)
+        {
+            Name = name;
+            _collection = new List<Song>();
+            _genres = new List<string>();
+        }
+
+        public string Name
+        {
+            get { return _name; }
+            set { _name = value; }
+        }
+       
+        public List<Song> Collection 
+        {
+            get { return _collection; }
+            set { _collection = value; }
+        }
+        public List<string> Genres 
```

Comment:
No need for private field _genres here. You are assigning the value to it and returning it.
Just return collection.Select(s => s.Genre).Distinct().ToList();
and remove the field

Reply:
Also leave line break between properties